### ISSUE TITLE
Tailing rack logs fails

### ISSUE THREAD

**Issue by bladealslayer** [convox/rack#1162]
Tailing the rack logs currently seems to be failing:

```
$ convox rack
Name    my-rack
Status   running
Version  20160827184744
Region   us-east-1
Count    2
Type     t2.small

$ convox logs -a my-rack
ERROR: my-rack not found

$ convox rack logs
ERROR: my-rack not found
```

**Comment by bladealslayer** [comment]
Looks like the CF stack naming changed and code assumes the rack stack is called `rack-name-rack-name`? https://github.com/convox/rack/blob/master/provider/aws/apps.go#L22

**Comment by ddollar** [comment]
Fixed in #1163
